node/src/Unison/Runtime/Html.hs

```diff
@@ -0,0 +1,26 @@
+module Unison.Runtime.Html where
+
+import Data.Maybe (listToMaybe, catMaybes, mapMaybe)
+import Data.Text (Text, toLower, pack)
+import Text.HTML.TagSoup
```

Comment:
Yeah, a style guide would be good. Is there already a Haskell style guide you roughly base your style on?